frontend/sass/components/_form.scss

```diff
@@ -141,3 +145,77 @@ input[type="range"]::-moz-range-thumb {
   width: 20px;
   border-radius: 50%;
 }
+
+.error-text {
+  color: map.get(map.get(abstracts.$highlight-colors, "pink"), "default");
+}
+
+.form {
+  padding: 25px;
+  background-color: abstracts.$primary-grey;
+  display: grid;
+}
+
+.form-submit {
+  font-weight: 600;
+  font-size: 1.25rem;
+  font-family: abstracts.$default-title-font;
+  margin-top: 25px;
+  margin-bottom: 25px;
+}
+
+.form-main {
+  display: grid;
+  justify-items: center;
+}
+
+.section-wrapper {
+  margin-top: 20px;
+  min-width: 50%;
+}
+
+.form-main-text {
+  display: flex;
+  justify-content: center;
+  font-weight: 600;
+  margin-top: auto;
+  padding-top: 10px;
+  padding-bottom: 10px;
+}
+
+.form-route-change {
+  display: grid;
+  grid-template-columns: auto auto auto;
+  justify-content: center;
+}
+
+.form-group-icon {
+  position: absolute;
+  background-color: inherit;
+  right: 0;
+  top: 0;
+}
+
+.form-group-icon:hover {
+  background-color: inherit;
+}
+
+.form-group-icon:focus {
+  outline: none;
+}
+
+.form-group {
+  position: relative;
+}
+
+.form-route-change-link-style {
```

Comment:
Tämän voisi laittaa form-route-changen sisään ettei tarvitse määritellä linkille erikseen classia, näin:
```
.form-route-change {
  display: grid;
  grid-template-columns: auto auto auto;
  justify-content: center;

  a {
    text-decoration: underline;
    }
}
```